lib/middleware/errors.js

```diff
@@ -47,8 +44,33 @@ function onUncaughtException(error) {
   setImmediate(process.exit.bind(null, 66), 1000);// eslint-disable-line no-undef
 }
 
+function renderView(err, req, res, next) {
+  const statusCode = err.status || 500;
+  const errorMessages = getErrorMessage(statusCode);
+
+  const viewData = {
```

Comment:
We do need them. We probably forgot to add them. The footer also has a default set of values so that's why it worked w/o but in order to make sure it's consistent across the app, we need to add them.